modules/data_release/ajax/GetFile.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * Controls access to data release files.
+ *
+ * PHP Version 5
+ *
+ *  @category Loris
+ *  @package  Data_Release
+ *  @author   Justin Kat <justinkat@gmail.com>
+ *  @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ *  @link     https://github.com/aces/Loris
+ */
+
+$user =& User::singleton();
+// Load config file and ensure paths are correct
+set_include_path(
+    get_include_path() . ":" .
+    __DIR__ . "/../project/libraries:" .
+    __DIR__ . "/../php/libraries"
+);
```

Comment:
This looks like old code that should be replaced by requiring the autoload file.